src/styles/layout/media-object.less

```diff
@@ -138,7 +138,7 @@ components/media-objects.less
     padding-bottom: @base-spacing-unit * 1/2;
     padding-right: @base-spacing-unit * 1/2;
 
-    .dropdown-menu.down {
+    > .dropdown-menu.down {
```

Comment:
#nit Can we add an `&` before each of these? So it will read: `& > .dropdown-menu.down {`